cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds

```diff
@@ -152,7 +152,7 @@ var __AdyenHelper : Object = {
     },
 
     getCheckoutUrl : function () : String {
-        var checkoutUrl = this.getLoadingContext() + "sdk/2.0.0/adyen.js";
+        var checkoutUrl = this.getLoadingContext() + "sdk/2.1.0/adyen.js";
```

Comment:
Just double checking if you have done all the necessary changes moving to v. 2.1.0 based on the list I've sent you